backend/src/v5/routes/teamspaces/projects/containers/containers.js

```diff
@@ -184,6 +203,94 @@ const establishRoutes = () => {
 	 */
 	router.get('/:container/stats', hasReadAccessToContainer, getContainerStats);
 
+	/**
+	 * @openapi
+	 * /teamspaces/{teamspace}/projects/{project}/containers/favourites:
+	 *   patch:
+	 *     description: Add containers to the user's favourites list
+	 *     tags: [Containers]
+	 *     operationId: appendContainers
+	 *     parameters:
+	 *       - teamspace:
+	 *         name: teamspace
+	 *         description: name of teamspace
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+   	 *       - project:
+	 *         name: project
+	 *         description: ID of project
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+	 *     requestBody:
+     *       content:
+     *         application/json:
+     *           schema:
+     *             type: object
+     *             properties:
+     *               containers:
+     *                 type: array
```

Comment:
![image](https://user-images.githubusercontent.com/11945337/133067839-6d741ae0-6f48-426c-938e-b580cabf9476.png)

schema isn't complete here. you should define what the members of array should be (i.e. strings)

![image](https://user-images.githubusercontent.com/11945337/133067866-c084f10b-68cf-4212-9fb0-9ea6ee443739.png)

Also define an example so the doc reader knows what to put in